GOVERNANCE.md

```diff
@@ -0,0 +1,228 @@
+- [Meetings](#meetings)
+- [Code of Conduct](#code-of-conduct)
+- [Security Response Team](#security-response-team)
+- [Modifying this Charter](#modifying-this-charter)
+- [Attribution](#attribution)
+<!-- TOC end -->
+
+## Values
+
+The RIOT community embraces the following values:
+
+* **Openness:** Communication and decision-making happens in the open and is discoverable for future reference.
+  As much as possible, all discussions and work events eventually take place in public forums and open repositories.
+
+* **Fairness:** All stakeholders have the opportunity to provide feedback and submit contributions, which will be considered on their merits.
+
+* **Community over Product or Company:** Sustaining and growing our community takes
+  priority over shipping code or sponsors' organizational goals.
+  Each contributor participates in the project as an individual.
+
+* **Inclusivity:** We innovate through different perspectives and skill sets, which can only be accomplished in a welcoming and respectful environment.
+
+* **Participation:** Responsibilities within the project are earned through participation, and contributors can grow into more responsible positions.
+
+
+## Community Processes
+The community around RIOT gathers many IoT developers and users from around the world, from the industry, from academia, and hobbyists.
+The RIOT community is open to everyone.
+Our channels to communicate can be found in our [contributing guidelines].
+The community self-organizes using the roles described below.
+
+
+## Roles
+### Contributors
+Contributors are people who contribute their work to RIOT.
+This includes,
+
+- of course, code contributions, but also
+- writing documentation,
+- bug reports,
+- other issue reports,
+- reviewing PRs,
+- participation in technical as well as non-technical discussions, or
+- organizational considerations.
+
+Code contributions are very welcome.
+In order to streamline and harmonize code quality, contributors must follow the [contributing guidelines].
+
+Contributors may be associated with organizations—by employment or otherwise—who have a vested interest in RIOT or may be individuals who have their own personal stakes in RIOT.
+We call these organizations and individuals “stakeholders” throughout this document to summarize them.
+
+### Maintainers
+
+Among contributors, some have maintainer status, which consists in rights (write access to the [RIOT GitHub repository](https://github.com/RIOT-OS/RIOT/)) and duties.
+The current maintainers can be found in the [maintainers list].
+
+Maintainers are people who care about RIOT and want to help it grow and improve.
+A maintainer is not just someone who can make changes, but someone who has demonstrated their ability to collaborate and organize with the team, get the most knowledgeable people to review code or documentation, contribute high-quality code or documentation, as well as follow through to fix issues (in code, tests, or tests).
```

Comment:
other .md files [CoC](https://raw.githubusercontent.com/RIOT-OS/RIOT/refs/heads/master/CODE_OF_CONDUCT.md) seem to have some line length limit applied  (maybe 100 with url exception)

But you are probably just saving this for the squash